tests/integration-tests/configs/pcluster3.yaml

```diff
@@ -6,25 +6,39 @@ test-suites:
       dimensions:
         - regions: ["af-south-1"]
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
-          oss: {{ common.OSS_ONE_PER_DISTRO }}
+          oss: ["alinux2"]
```

Comment:
why this change?

Reply:
Just changed it in pcluster3.yaml. I have been trying to use pcluster3.yaml to do some quick verification. This change is to reduce the execution time of the test.